media/css/custom.scss

```diff
@@ -0,0 +1,496 @@
+  border: 1px solid $grayish-blue;
+  border-width: 0 0 1px 1px;
+  font-size: 20px;
+}
+
+th {
+  text-align: center;
+  vertical-align: middle;
+  position: relative;
+}
+
+th {
+  font-weight: bold;
+  font-size: 14px;
+}
+
+td:hover,
+th:hover {
+  background-position: 0 -81px;
+  color: $light-black;
+}
+
+.events {
+  position: relative;
+  ul {
+    text-align: left;
+    position: absolute;
+    display: none;
+    z-index: 1000;
+    padding: 15px;
+    background: $light-grayish-blue;
+    color: $white;
+    border: 1px solid $white;
+    font-size: 15px;
+    width: 600px; // 200px, wow banner
+    @include border-radius(3px);
+    list-style: none;
+    color: $gray;
+    -webkit-box-shadow: 0 8px 8px $dark-gray;
+    box-shadow: 0 8px 8px $dark-gray;
+  }
+  li {
+    padding-bottom: 5px;
+    span {
+      display: block;
+      font-size: 12px;
+      text-align: justify;
+      color: $gray;
+      &.title {
+        font-weight: bold;
+        color: $light-black;
+      }
+    }
+  }
+}
+
+// applicants help
+
+.table-help-inf {
+  font-size: 13px;
+  width: 572px;
+  border-collapse: collapse;
+  margin-top: 30px;
+  border: 1px solid;
+  th {
+    border: 1px solid;
+  }
+  td {
+    border: 1px solid $light-gray;
+    padding-left: 3px;
+  }
+}
+
+.table-help {
+  text-align: center;
+}
+
+// student council
+
+.i2Style {
+  font: bold 24px Tahoma, Geneva, sans-serif;
+  color: $white;
+  border: 0 solid $white;
+  text-shadow: 0 -1px 1px $light-black;
+  box-shadow: 1px -12px 0 $moderate-blue;
+  -moz-box-shadow: 1px -12px 0 $moderate-blue;
+  -webkit-box-shadow: 1px -12px 0 $moderate-blue;
+  width: 575px;
+  padding: 10px;
+  cursor: pointer;
+  margin: 0 auto;
+  &:active {
+    cursor: pointer;
+    position: relative;
+    top: 2px;
+  }
+  ~ img {
+    margin-bottom: 35px;
+  }
+}
+
+.widget-item {
+  height: 180px;
+  padding-right: 0;
+  padding-bottom: 0;
+  padding-left: 0;
+  position: relative;
+  opacity: .75;
+  transition: opacity ease .5s;
+  -o-transition: opacity ease .5s;
+  -ms-transition: opacity ease .5s;
+  -moz-transition: opacity ease .5s;
+  -webkit-transition: opacity ease .5s;
+  &:hover {
+    opacity: 1;
+  }
+  a {
+    display: block;
+    height: 100%;
+    width: 100%;
+    position: relative;
+    text-decoration: none;
+    font-size: 22px;
+    & > div {
+      position: absolute;
+      padding: 16px 16px 20px;
+      bottom: 0;
+      left: 0;
+      right: 0;
+    }
+  }
+  &.blue {
+    a {
+      & > div { // .widget-item.blue a > div
+        background-color: $widget-link-highlight;
+        color: $white;
+      }
+    }
+  }
+  &.schedule {
+    background: url("../img/schedule.jpg") center center;
+    background-size: cover;
+  }
+  &.announcement {
+    background: url("../img/announcement.jpg") center center;
+    background-size: cover;
+  }
+}
+
+#calendar {
+  max-width: 900px;
+  margin: 0 auto;
+}
+
+.link-box-graduate {
+  a {
+    color: $white;
```

Comment:
Properties should be ordered background, color, display, font-size, line-height, margin-bottom, padding, text-align, width